src/components/user-login/SignIn.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="px-6 py-6">
+    <form>
+      <v-row>
+        <v-col cols="12" class="mt-2 py-2">
+          <v-text-field
+            v-model="emailAddress"
+            :error-messages="vuelidate.emailAddress.$errors.map((e) => e.$message as string)"
+            label="Email"
+            required
+            @input="vuelidate.emailAddress.$touch"
+            @blur="vuelidate.emailAddress.$touch"
+          />
+        </v-col>
+      </v-row>
+      <v-row class="mt-6">
+        <v-col cols="12" class="py-2">
+          <v-text-field
+            v-model="password"
+            :error-messages="vuelidate.password.$errors.map((e) => e.$message as string)"
+            label="Password"
+            required
+            type="password"
+            @input="vuelidate.password.$touch"
+            @blur="vuelidate.password.$touch"
+          />
+        </v-col>
+      </v-row>
+
```

Comment:
As per the requirement, we need to provide a 'forgot password' link in the Login Page

Reply:
Addressed in [481eadd](https://github.com/mpieris/resume-genius-ui/pull/1/commits/481eadd418140a1612aef7adc2f8b219a2e42611)